DEVNOTES.md

```diff
@@ -18,18 +18,21 @@ environment by looking at the deployed configs in https://duos-k8s.dsde-{%ENV%}.
 `dev`. Certain features are available only in specific environments. Setting the `env` value to the desired environment 
 will simulate it for local development. The installation steps outlined in this step can also be completed using the 
 [render-configs.sh](scripts/render-configs.sh) script which can generate all required files for local development.
+
```

Comment:
These additional spaces are added because not all markdown parsers recognize the triple backticks directly after a string.